### DISCUSSION TITLE
How to run with playwright?

### DISCUSSION THREAD

**Discussion by ospfranco** [blitz-js/blitz#3118] [Q&A]
Hi, I wanted to start writing some playwright tests, I found [this next.js tutorial](https://frontend-digest.com/using-playwright-to-test-next-js-applications-80a767540091) and I think it also makes sense for blitz.

The basic idea is to do a production transpiling of the app when setting up the test environment, there is however one step I'm not sure how to resolve, which is building the app via cli command:

```js
// tests/global-setup.ts
const path = require("path");
const cli = require("next/dist/cli/next-build");
async function globalSetup() {
  process.env.PLAYWRIGHT = "1";
  if (process.env.SKIP_BUILD === "1") {
    console.log("skipping build as SKIP_BUILD is set");
  } else {
    await cli.nextBuild([path.join(__dirname, "..")]);
  }
}
module.exports = globalSetup;
```

Is there any way to do the same for blitz?

**Comment by beerose** [comment]
It actually should work. Does it cause some errors?
